test/e2e/failover_test.go

```diff
@@ -139,6 +143,130 @@ var _ = ginkgo.Describe("failover testing", func() {
 			})
 		})
 	})
+
+	ginkgo.Context("Deployment propagation testing", func() {
```

Comment:
We have two `Deployment propagation testing` testing? I think you should highlight your case here.

Reply:
Here is "CustomResourceDefinition propagation testing"